.github/workflows/assets.yml

```diff
@@ -12,21 +15,9 @@ jobs:
     runs-on: ubuntu-20.04
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 2.7
-      # Get OS version - https://github.com/actions/cache/issues/543
-      - run: echo "OS_VERSION=`lsb_release -sr`" >> $GITHUB_ENV
-      - uses: actions/cache@v2
-        with:
-          path: vendor/bundle
-          key: ${{ runner.os }}-${{ env.OS_VERSION }}-gems-pr-${{ hashFiles('**/Gemfile.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-${{ env.OS_VERSION }}-gems-pr-
+          bundler-cache: true
       - name: Build assets
-        run: |
-          gem install bundler -v 2.4.22
```

Comment:
This will cause it to always use the default for the ruby version I think?

Reply:
In theory if the bundler version that comes with the ruby version is new enough it should read the BUNDLED WITH in the Gemfile and use that